orttraining/orttraining/python/training/ortmodule/_custom_gradient_registry.py

```diff
@@ -109,6 +109,34 @@ def max_pool2d_gradient():
     ]
 
 
+def minmax_gradient():
```

Comment:
Better add comment to explain PyTorch's gradient logics are different for these two cases, so we also separate handling these cases.